app/src/main/res/drawable-v24/outline_edittext.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
+<shape xmlns:android="http://schemas.android.com/apk/res/android"
+
+    android:thickness="0dp"
+
+    android:shape="rectangle">
+    <stroke android:width="3dp"
+
+        android:color="@color/colorPrimary"/>
+    <corners android:radius="5dp" />
+</shape>
```

Comment:
You don't need to do this, theres a style available that you have to just specify in attributes of the view in XML